app/src/main/res/values/strings.xml

```diff
@@ -3,4 +3,13 @@
     <string name="email">Email</string>
     <string name="password">Hasło</string>
     <string name="login">Zaloguj</string>
+    <string name="register">Zarejestruj się</string>
+    <string name="zarejestruj_sie">Zarejestruj się</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```

Comment:
można usunąć "hello_blank_fragment" a w reszczie prosiłbym o angoielkie nazwy bo jak się wpiszę jakiś Polski znak w name="<tutaj>" to może się wywalać apka lub budowanie apki